services/app-api/serverless.yml

```diff
@@ -14,7 +14,8 @@ plugins:
   - serverless-associate-waf
   - serverless-offline
   - serverless-stack-termination-protection
-  - serverless-enable-api-logs
+  - serverless-idempotency-helper
+  - serverless-iam-helper
```

Comment:
This is how you enable the serverless-iam-helper plugin.  It takes its input from any values specified at provider.iam.role.path and provider.iam.role.permissionsBoundary.